### PR TITLE
Copy doc files to src/doc to avoid sym links

### PR DESCRIPTION
**Pull request by dmarcos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,14 +39,15 @@
   },
   "scripts": {
     "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && multidep multidep.json",
+    "copyDocs": "node scripts/copyDocs.js multidep.json",
     "clean": "hexo clean",
     "deploy": "hexo deploy && open 'https://aframe.io/'",
-    "dev": "npm run clean && npm run generate:config && npm run server",
+    "dev": "npm run clean && npm run copyDocs && npm run generate:config && npm run server",
```
Comment (ngokevin) [inline package.json]: I would add copyDocs as a part of postInstall and bumpdocs, rather than dev

File: package.json [hunk 2]
```diff
@@ -38,14 +39,15 @@
...
     "generate": "npm run generate:config && hexo generate",
     "generate-awoa": "node scripts/generateAWeekofAFrame.js",
-    "generate:config": "yaml-cat '_config.prod.yml' '_config.local.yml' -n -m > _config.yml",
+    "generate:config": "yaml-cat _config.prod.yml _config.local.yml -n -m > _config.yml",
     "predeploy": "cp _config.prod.yml _config.yml && npm run clean",
-    "postinstall": "multidep multidep.json",
```
Comment (dmarcos) [inline package.json]: sorry mistake

File: scripts/copyDocs.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          console.error('Copy failed: ' + error);
+      } else {
+          console.info('Copied ' + results.length + ' doc files');
+      }
+    });
+  }
+});
+
+
+// function getSpec(specPath) {
```
Comment (ngokevin) [inline scripts/copyDocs.js]: Needed?

File: scripts/copyDocs.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+var copy = require('recursive-copy');
+var fs = require('fs');
+var specPath = process.argv[2];
+
+fs.readFile(specPath, 'utf8', function (err, data) {
```
Comment (ngokevin) [inline scripts/copyDocs.js]: Sprinkle some comments just for my future sake

File: package.json (hunk 1 above)
Comment (dmarcos) [inline package.json]: ok

File: package.json [hunk 5]
```diff
@@ -37,15 +38,16 @@
     "version": "3.2.2"
   },
   "scripts": {
-    "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && multidep multidep.json",
+    "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && npm run copy-docs && smultidep multidep.json",
```
Comment (ngokevin) [inline package.json]: typo at the end: `smultidep`. The copy docs should be run after the install. also it got hyphenated which isn't reflected in the `dev` script (which should be removed) and inconsistent with JS.

File: package.json [hunk 6]
```diff
@@ -37,15 +38,16 @@
...
     "deploy": "hexo deploy && open 'https://aframe.io/'",
-    "dev": "npm run clean && npm run generate:config && npm run server",
+    "dev": "npm run clean && npm run copyDocs && npm run generate:config && npm run server",
     "generate": "npm run generate:config && hexo generate",
     "generate-awoa": "node scripts/generateAWeekofAFrame.js",
-    "generate:config": "yaml-cat '_config.prod.yml' '_config.local.yml' -n -m > _config.yml",
+    "generate:config": "yaml-cat _config.prod.yml _config.local.yml -n -m > _config.yml",
     "predeploy": "cp _config.prod.yml _config.yml && npm run clean",
-    "postinstall": "multidep multidep.json",
+    "postinstall": "npm run copy-docs && multidep multidep.json",
```
Comment (ngokevin) [inline package.json]: also here, copy should be run after

File: package.json [hunk 7]
```diff
@@ -37,15 +38,16 @@
     "version": "3.2.2"
   },
   "scripts": {
-    "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && multidep multidep.json",
+    "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && npm run copy-docs && pomultidep multidep.json",
```
Comment (ngokevin) [inline package.json]: pomultidep?

Comment (ngokevin) [inline package.json]: multidep has to be run before copydocs

File: package.json [hunk 8]
```diff
@@ -37,15 +38,16 @@
...
+    "copy-docs": "node scripts/copyDocs.js multidep.json",
```
Comment (ngokevin) [inline package.json]: `copydocs` (i should rename the generate-awoa to generateawoa too)

**Comment by ngokevin** [comment]
I get `SyntaxError: Unexpected token u` because Hexo is trying to run the script since it does that to all scripts in the `scripts/` directory.

Let's move those scripts out (with generate-awoa script) to like `node_scripts/`.

**Comment by ngokevin** [comment]
Everyone works for me once I do ^

**Comment by dmarcos** [comment]
@ngokevin I merged this with the new post but I can't deploy. I think there's still something missing with the routes of the `docs`. This is the error I get Error Docs

File: src/_posts/screenshot.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+---
+title: A-Frame Tips - Screenshot
+author: twitter|dmarcos|Diego Marcos
+date: 2016-12-23 10:00:00
+layout: blog
+---
+
+<script src="https://aframe.io/releases/0.4.0/aframe.min.js"></script>
+
+Did you know that [A-Frame 0.4.0](http://localhost:4000/blog/aframe-v0.4.0/) lets you take a screenshot of any scene? Press `<alt> + <ctrl> + s` to take a regular screenshot or `<alt> + <ctrl> + <shift> + s` to capture an equirectangular panorama. The latter is very handy to create scene previews so users can take a quick peek without having to wait for the whole experience to load. You could create a gallery of content that presents a grid of interactive previews without the overhead of initializing every single scene. This is what it takes to create a preview of the A-Frame [Anime UI example](https://aframe.io/aframe/examples/showcase/anime-UI/) based on an image generated with `<alt> + <ctrl> + <shift> + s`:
```
Comment (ngokevin) [inline src/_posts/screenshot.md]: Localhost

Comment (dmarcos) [inline src/_posts/screenshot.md]: Do you mean the image reference by `a-sky`?

**Pull request merged**